eprihlaska/consts.py

```diff
@@ -6,8 +6,9 @@
 
 DIR = os.path.dirname(os.path.abspath(__file__))
 
-SELECT_STUDY_PROGRAMME = _('Vyberte si, prosím, aspoň prvý študijný program')
-NO_ACCESS_MSG = _('Nemáte oprávnenie pre prístup k danému prístupovému bodu')
+SELECT_STUDY_PROGRAMME = _('Vyberte si, prosím, aspoň prvý študijný program') # noqa
+SELECT_STUDY_PROGRAMME_ONCE = _('Vyberte si, prosím, študijný program najviac jeden krát') # noqa
```

Comment:
@tvinar I am not really sure what the proper spelling would be, so I'd be very happy to take any suggestions :slightly_smiling_face:

Reply:
A small ping @tvinar :slightly_smiling_face: